api/azure-pipelines.ts

```diff
@@ -1,18 +1,136 @@
-import cheerio from 'cheerio'
```

Comment:
`cheerio` need to be imported

Reply:
The token is not mandatory. The token is required for private projects.